README.md

```diff
@@ -85,7 +85,7 @@
 * satohchen
 * shinyzhu
 * suave
-* sxzz
+* 三咲智子
```

Comment:
We don't need to update `README.md`. Instead, it is generated by `README.tpl`. I prefer to add a file `sxzz.yaml` into https://github.com/opensource-f2f/episode/tree/master/archives

Then, you could add your Chinese name like this:
```yaml
name: 三咲智子
```

Of course, we need to support to show it at https://github.com/opensource-f2f/website